src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,38 @@
+$color-light-blue: rgba(0, 172, 220, 1);
+
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__item {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: inline-block;
+    text-decoration: none;
+    font-size: 12px;
+    text-transform: uppercase;
+    font-weight: 500;
+    line-height: 60px;
+    color: rgba(6, 11, 53, 1);
+    transition: all 0.3s ease-in-out;
+
+    &:hover {
+      color: #{$color-light-blue};
+    }
+  }
+}
+
+.is-active {
+  color: #{$color-light-blue};
+
+  &::after {
+    content: "";
+    display: block;
+    border-bottom: 4px solid #{$color-light-blue};
+    border-radius: 8px;
+  }
+}
```

Comment:
It is such a bad idea to create a line under the link like that. Try to rewrite it with the position property and top/right/bottom/left properties. In that way, we can underline not only the first link. Also, the next links could be underlined